src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,11 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[3];
+
+        for (int i = 0; i < balls.length;) {
+            balls[i] = new Lottery().getRandomBall();
+            System.out.println(balls[i]);
+        }
```

Comment:
The loop does not increment the index variable `i`, which results in an infinite loop. You should increment `i` in each iteration to ensure the loop progresses and initializes all elements of the `balls` array.